IMC_WebDev/FinalBlog.md

```diff
@@ -0,0 +1,224 @@
+          <div class = "wall3"></div>
+          <div class = "wall4"></div>
+          <div class = "roof2"></div>
+        </div>
+        <div class = "road1"></div>
+        <div class = "road2"></div>
+      </div>
+    
+    	<div class = "ocean"></div>
+    	<div class = "ramp1"></div>
+      <div class = "ramp2"></div>
+      <div class = "attempt1"></div>
+    
+      <div class = "bridge">
+      	<div class = "railing1"></div>
+      </div>
+      
+      <div class = "railing2"></div>
+      
+    
+    	<div class = "waterfall"></div>
+    
+    	<div class = "ground2">
+    		<div class = "grassArea"> <!--GRASS AREA-->
+    			<div class = "treeTrunks"></div> 
+          <div class = "triangleTops"></div>
+          <div class = "circleTops"></div> 
+    		</div>
+        <div class = "bottomFront"></div> 
+    	</div>
+      
+      <div class = "car">
+        <div></div>
+        <div></div>
+      </div>
+    
+    	<div class = "boat">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+    
+    	<div class = "sun"></div>
+  </body>
+```
+
+This looks like a huge chunk of code right now, but let’s break it down!
+
+HTML works as the canvas for all of our objects, so for every object in our city, we create a div class for it. Once we do that, we can edit those divs using our CSS, which is like our paint! It allows us to control how the objects appear on the site, like color, layout, etc.
+
+Sometimes, objects have “children." For example, since our buildings are placed on the ground, the buildings are the “children” of the ground.
+You can see that “building1” is placed within the “ground” tag in our code.
+
+Our “building1” also has children of its own. Since walls and a roof are components of our building, we place them inside our &lt;building1&gt; tag. We repeat this whenever components belong to each other. (You can also see this with the grass area and trees.)
+
+After we create all of our objects and their “children,” it is time to close our body and our html tag by using /body and /html.
+
+When it’s time to design our objects in CSS, we MUST use the exact name we created for them in our HTML. For example, if I want to change the color of the bridge in my city, I need to name it “bridge” in my CSS, just like how it is in my HTML.
+
+Next, let’s prepare our editor for the CSS portion of the code. We do this by establishing a style tag. Since we will be using CSS, our style tag will look like this: 
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>Mini City Webinar</title>
+    	<style TYPE="text/css">
+```
+
+
+We will be writing all of our CSS in the head portion of our code.
+
+To style our webpage, we will need to learn the CSS syntax. A CSS rule-set consists of a selector and a declaration block.
+<img src=https://lh3.googleusercontent.com/3UcaVdjNU8MuMn4Feims4asq1epCgIOEwJX2goOH8SZE2eP2IKz5vRvCuSL9howvCc_Co_G2TE8Tqe5pIPgTAITV9KB28hqPEWXLzvZZ7xTnF1GNBohcY75WK00KBHIfbJ4nufVR>
```

Comment:
If you could add a sentence or two (or maybe even revise the sentences you already have) to reference/explain this useful image a bit more to the audience, I think that would help them better understand the CSS rule-set